template/ios/Podfile

```diff
@@ -25,12 +30,6 @@ target 'HelloWorld' do
     # Pods for testing
   end
 
-  # Enables Flipper.
-  #
-  # Note that if you have use_frameworks! enabled, Flipper will not work and
-  # you should disable the next line.
-  use_flipper!()
```

Comment:
It won't work, the function `use_flipper` won't exist anymore after this change.
The only thing is probably how tell the user that the flipper configuration is now into the `use_react_native` function.
In case we wanted to do so we could use https://docs.ruby-lang.org/en/2.6.0/Gem/Deprecate.html